docs/content/latest/develop/graphql/hasura.md

```diff
@@ -29,64 +29,34 @@ If you're using `yb-ctl` to start your cluster, you can add the option like this
 $ ./bin/yb-ctl start --tserver_flags "ysql_suppress_unsupported_error=true"
```

Comment:
minor: would be good to suggest the docker usage here because the hasura server below uses docker anyway.